quetz/config.py

```diff
@@ -113,6 +113,13 @@ class Config:
             "session",
             [ConfigEntry("secret", str), ConfigEntry("https_only", bool, default=True)],
         ),
+        ConfigSection(
+            "local_store",
+            [
+                ConfigEntry("redirect_enabled", bool, default=False),
+                ConfigEntry("redirect_endpoint", str, default="/files"),
```

Comment:
We could maybe only define one option: if redirect_endpoint is set to an empty string, redirect would be disabled.
But I think it's a bit clearer like that.